apps/web/components/apps/AppPage.tsx

```diff
@@ -98,6 +99,11 @@ export const AppPage = ({
    * which is caused by heavy queries in getServersideProps. This causes the loader to turn off before the page changes.
    */
   const [isLoading, setIsLoading] = useState<boolean>(mutation.isPending);
+  const enabledOnTeams = doesAppSupportTeamInstall({
```

Comment:
nit: `enabledOnTeams` make it sound like user has it enabled.
```suggestion
  const availableForTeams = doesAppSupportTeamInstall({
```